workflow-manager/batchpath/batchpath.go

```diff
@@ -40,6 +55,22 @@ func (bpl List) Swap(i, j int) {
 	bpl[i], bpl[j] = bpl[j], bpl[i]
 }
 
+// WithinInterval returns the subset of the batches in the receiver that are
+// within the given Interval.
+func (bpl List) WithinInterval(interval wftime.Interval) []string {
```

Comment:
I would probably move this to time.go module you made. It seems like it would make sense to have something like this on the Interval struct

Reply:
So something like `func (i *Interval) BatchesWithin(list batchpath.List) []string`? Sure, sounds reasonable!

Reply:
I think more so `func (i *Interval) TimeWithin(time time.Time)` but that works too

Reply:
That's even better! I will incorporate that suggestion.